plugins/synaptics-mst/fu-synaptics-mst-device.c

```diff
@@ -1269,8 +1269,29 @@ fu_synaptics_mst_device_update_firmware(FuSynapticsMstDevice *self,
 {
 	g_autoptr(FuSynapticsMstDeviceHelper) helper = fu_synaptics_mst_device_helper_new();
 
-	helper->fw =
-	    fu_bytes_new_offset(fw, 0x0, fu_device_get_firmware_size_max(FU_DEVICE(self)), error);
+	guint32 fwSize;
```

Comment:
wrong case, maybe `gsize truncate_size = 0` or something like that?

Reply:
Do we need that if return FALSE?

Reply:
I don't know if coverity is clever enough to catch that in the switch statement. I think it's probably fine if you leave out the `= 0`